### DISCUSSION TITLE
Sites com páginas offline criativas

### DISCUSSION THREAD

**Discussion by willianjusten** [frontendbr/forum#1006] [Discussão]
Fala pessoal, eu lancei um post hoje sobre como fiz para o meu blog funcionar offline para alguns posts/páginas e redirecionar para um joguinho caso a página que a pessoa tentou acessar não esteja cacheada.

O post é [Como fazer seu site funcionar offline com PWA](https://willianjusten.com.br/como-fazer-seu-site-funcionar-offline-com-pwa/).

E aí me surgiu uma curiosidade, vocês conhecem soluções legais usando Service Workers para  páginas offline? Eu adoraria ver =D

**Comment by NandoSangenetto** [comment]
Se liga na página offline do https://dev.to

Eu achei bem bacana e bem humorada, porém você tem que conhecer bem seu público para uma abordagem assim.

A princípio só a página offline deles me chamou atenção.

**Comment by willianjusten** [comment]
Bacana a página 404 deles, a minha [é assim](https://willianjusten.com.br/404)

**Comment by gabrieldarezzo** [comment]
A melhor sempre vai ser essa:

http://www.pt.org.br/essapaginanaoexiste

husasahusaus

**Comment by willianjusten** [comment]
Essa é genial @gabrieldarezzo ! ahuahua

**Comment by lnfnunes** [comment]
Ae pessoal, aproveitando a discussão, deixo [um repo](https://github.com/lnfnunes/404-PageNotFound) que criei há um tempo onde listo diversas **_páginas 404_** bem iradas!
Quem tiver alguma sugestão que ainda não esteja na lista, por favor mandem o PR lá: https://github.com/lnfnunes/404-PageNotFound

Uma ideia legal seria essa lista de _offline pages_, hein @willianjusten !?
